src/lib.rs

```diff
@@ -321,14 +321,17 @@ impl FileDialog {
   }
 
   fn ui(&mut self, ctx: &Context, is_open: &mut bool) {
-    let (align, offset) = self.anchor;
     let mut window = Window::new(RichText::new(self.title()).strong())
       .open(is_open)
       .default_size(self.default_size)
-      .anchor(align, offset)
       .resizable(self.resizable)
       .collapsible(false);
 
+    if self.anchor != None {
```

Comment:
Please change this to `if let Some((align, offset)) = self.anchor {`

Reply:
Done